homeassistant/components/switch_as_x/__init__.py

```diff
@@ -32,6 +32,13 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         )
         return False
 
+    # Hide the wrapped entry if registered
```

Comment:
Should we do this inside the config flow? That way a user might decide to undo the action.

Reply:
Moved to inside the config flow by f0ecede